offchain-modules/packages/x/src/ckb/tx-helper/generated/ckb_lock_cell_data.ts

```diff
@@ -0,0 +1,10 @@
+import Script = CKBComponents.Script;
+
+export interface CkbLockCellData {
```

Comment:
1. refactor type define to other direction, generated direction is used for code generated by molecule
2. better renamed it to `NervosLockAssetTxMetaData` according to its meaning

Reply:
renamed to NervosLockAssetTxMetaData and moved to ../db/model